.github/workflows/artifact-publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Publish Java Package
```

Comment:
What will be the approx. artifact size and where we are using this ? As our ODE GitHub org is on free plan we have artifact storage limit of 500mb.

Reply:
Approx:
jpo-ode-common.jar - 38 KB
jpo-ode-core.jar - 154 KB
jpo-ode-plugins.jar - 436 KB
jpo-ode-svcs.jar - 134 **MB** (large because it includes the Spring web stuff)

I'm not certain, but this seems to be saying that the storage limits for maven packages don't apply to public repos:
https://docs.github.com/en/packages/learn-github-packages/introduction-to-github-packages

I am fairly certain that there is no time limit on Maven packages (which are not the same as Gitub Action build artifacts)

Reply:
I checked, and the normal jar for jpo-ode-svcs is approx 219KB, so if we changed the action to publish that, the total space would be less than 1MB for everything

Reply:
@SaikrishnaBairamoni Here is the relevant [PR](https://github.com/CDOT-CV/jpo-ode/pull/102) that addresses this issue. We will work on getting this update merged ASAP.